plasmapy/plasma/equilibria1d.py

```diff
@@ -0,0 +1,97 @@
+__all__ = ["HarrisSheet"]
+
+import astropy.constants as const
+import astropy.units as u
+import math
+import numpy as np
+
+
+class HarrisSheet:
+
+    """
+    Define a Harris Sheet Equilibrium.
+
+    Parameters
+    ----------
+    B0 : `~astropy.units.Quantity` 
+         magnetic field.
+
+    delta : 'float'
+        Delta is the thickness of the sheet.
+
+    P0 : 'float'
+        Plasma Pressure.
+
+    Notes
+    -----
+    A current sheet is current limited to a surface.
+
+    Examples
+    --------
+    >>> harris_sheet = HarrisSheet(delta = 3*u.m ,B0 = 2*u.T)
+    >>> harris_sheet.magnetic_field(y = 5*u.m)
+    <Quantity 1.8622... T>
+
+    """
+
+    def __init__(self, B0, delta, P0=0 * u.Pa):
+        self.B0 = B0
+        self.delta = delta
+        self.P0 = P0
+
+    def magnetic_field(self, y):
+        r"""
+        Compute the magnetic field along y = 0.
+
+        This equation uses the asymptotic magnetic field strength along with y=0.
+
+        Delta provides the thickness of the sheet.
+
+        .. math::
+
+            B_x(y) = B_0 \tanh \left( \frac{y}{\delta} \right)
+
+        Parameters
+        ----------
+
+        y : `float`
+          The axis of reference.
+
+        """
+        return self.B0 * np.tanh(y / self.delta)
```

Comment:
```suggestion
        return self.B0 * np.tanh(u.rad * y / self.delta)
```
Apparently, `astropy.units` is set up so that (hyperbolic) trigonometric functions require that the arguments be in radians...